### PR TITLE
Update istio 1.2 performance benchmarking docs

### PR DESCRIPTION
**Pull request by carolynhu**
This PR is to change this blog post https://istio.io/blog/2019/performance-best-practices/ to reference the latest istio performance benchmark repo.

### CODE REVIEW

**Comment by carolynhu** [comment]
@geeknoid Need approval for this PR. Thanks.

File: content/en/blog/2019/performance-best-practices/index.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: "Best Practices: Benchmarking Service Mesh Performance"
 description: "Tools and guidance for evaluating Istio's data plane performance."
 publishdate: 2019-07-09
+lastupdatedate: 2019-09-05
```
Comment (carolynhu) [inline content/en/blog/2019/performance-best-practices/index.md]: @geeknoid Do you mean add this? I'm not sure which key-name I should use here.

Comment (geeknoid) [reply content/en/blog/2019/performance-best-practices/index.md]: Nope, it's `last_update`.

See <https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#updating-front-matter> for all the front-matter fields supported.

**Comment by mandarjog** [comment]
@carolynhu was the previous commit pointed to be the blog post not working? 
Why do you want to update it ? As long as the blog post is consistent with the version numbers and commits we should not change it.

We are updating the actual data in the concepts/performance doc, that should be sufficient.

**Comment by carolynhu** [comment]
@mandarjog Since recently we updated the README docs, the old README is not working. So I update the commit to the working one. But the results should not update right? Then we do not need the `last_update` label? Since in this PR I only want the reader to have the working README doc to follow up.

**Pull request merged**